src/frontend/components/DashboardThumbnail/index.tsx

```diff
@@ -95,7 +95,7 @@ const BaseDashboardThumbnail = (props: BaseDashboardThumbnailProps) => {
     try {
       setDisableUploadBtn(true);
       if (props.thumbnail === null) {
-        props.addThumbnail(await createThumbnail(props.jwt));
```

Comment:
You don't need this props anymore, you can remove it.

Reply:
done